internal/commands/verify.go

```diff
@@ -128,6 +128,7 @@ func NewVerifyCommand(ctx context.Context) *cobra.Command {
 	cmd.Flags().StringP("output", "o", output.OutputStandard, fmt.Sprintf("Output format for conftest results - valid options are: %s", output.Outputs()))
 	cmd.Flags().Bool("junit-hide-message", false, "Do not include the violation message in the JUnit test name")
 
+	cmd.Flags().String("capabilities", "", "A json path that can restrict opa functionality against a given policy. Default: all operations allowed")
```

Comment:
nit: Consider rephrasing to `Path to JSON file that can ...`

Reply:
done, too